kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/KyuubiStatement.java

```diff
@@ -796,15 +796,15 @@ private void parseMetadata(
       metadataResp
           .getStatus()
           .getInfoMessages()
+          .stream()
+          .filter(hint -> Utils.isKyuubiOperationHint(hint))
           .forEach(
               line -> {
                 String[] keyValue = line.split("=");
                 if (keyValue.length == 2) {
                   String key = keyValue[0];
```

Comment:
since the pattern is CASE_INSENSITIVE, we need to normalize the key here, e.g. to lower/upper case, and the split should always reserve 2 parts, then assert is more suitable here.